src/plugins/legacy/itkINRDataImageReader/COPYING.txt

```diff
@@ -0,0 +1,10 @@
+Copyright (c) <YEAR>, <OWNER>
```

Comment:
is this file necessary? It could be removed, or the tags could be filled. Idem for README.txt

Reply:
agreed, should be removed, we already have this at the root folder or so

Reply:
done